docs/kubernetes/domains-in-google-cloud-infrastructure.md

```diff
@@ -0,0 +1,83 @@
+# Domains in Google Cloud infrastructure
+If you add a new domain, you also want to have this domain available on Google Cloud. This article describes how to perform such changes.
+
+To pass traffic through domain name you need to modify `ingress.yml` and `webserver-php-fpm.yml` which serves as webserver and listens on domain names.
+
+## Adding a new domain
+
+Let's say we want to listen to a new domain name.
+
+Open [ingress.yml](/project-base/kubernetes/ingress.yml) file and add new domain block into `spec -> rules`:
+
+```
+// kubernetes/ingress.yml
+
+rules:
+    -   host: ~
+        http:
+            paths:
+            -   path: /
+                backend:
+                    serviceName: webserver-php-fpm
+                    servicePort: 8080
+```
+
+Create secrets from SSL certificate in `ingress-patch.yaml` in `kubernetes/kustomize/overlays/production/ingress-patch.yaml`:
+```
+// kubernetes/kustomize/overlays/production/ingress-patch.yaml
+
+- name: domain-${DOMAIN_NUMBER}-ssl-certificates
+  commands:
+    tls.key: "cat ${ANOTHER_DOMAIN_SSL_DIRECTORY}/tls.key"
+    tls.crt: "cat ${ANOTHER_DOMAIN_SSL_DIRECTORY}/tls.crt"
+    ca.crt: "cat ${ANOTHER_DOMAIN_SSL_DIRECTORY}/ca.crt"
+```
+
+*Note: Replace ${DOMAIN_NUMBER} with a number of a domain*
+
+Next, add SSL certificates for new domain:
+
+```
+spec:
+    tls:
+    -   hosts:
+        secretName: domain-${DOMAIN_NUMBER}-ssl-certificates
+```
+
+Open [.ci/deploy-to-google-cloud.sh](/project-base/.ci/deploy-to-google-cloud.sh) file and set your new domain host to `ingress.yml` and `webserver-php-fpm.yml` host name:
+
+```
+// .ci/deploy-to-google-cloud.sh
+
+NEW_DOMAIN_HOST=${NEW_DOMAIN_HOST}
```

Comment:
shouldnt be this NEW_DOMAIN_NAME ???

Reply:
$NEW_DOMAIN_HOST is set to spec.rules[].**host** so i guess it is correct name.